astropy/coordinates/tests/accuracy/generate_ref_ast.py

```diff
@@ -60,12 +79,12 @@ def ref_fk4_no_e_fk4(fnout="fk4_no_e_fk4.csv"):
     t.add_column(Column(name="dec_fk4ne", data=dec_fk4ne))
     t.add_column(Column(name="ra_fk4", data=ra_fk4))
     t.add_column(Column(name="dec_fk4", data=dec_fk4))
-    f = open(os.path.join("data", fnout), "wb")
-    f.write(
-        f"# This file was generated with the {os.path.basename(__file__)} script, and"
-        " the reference values were computed using AST\n"
-    )
-    t.write(f, format="ascii", delimiter=",")
+    with open(DATA_DIR / fnout, "w") as fh:
```

Comment:
Does this mean that the script that claims to have produced reference data for our tests cannot possibly have done that?

Reply:
I'm not quite sure, it may have been able to run as is in Python 2, where strings and bytes objects behaved differently. Unfortunately I cannot test this (Python 2 doesn't compile on mac ARM).

Reply:
I'm not familiar with Python 2 either. I suppose it's plausible enough that this was Python 2 script that has not been properly ported to Python 3.